lib/events.js

```diff
@@ -257,8 +257,9 @@ function _addListener(target, type, listener, prepend) {
       if (m && m > 0 && existing.length > m) {
         existing.warned = true;
         const w = new Error('Possible EventEmitter memory leak detected. ' +
-                            `${existing.length} ${type} listeners added. ` +
-                            'Use emitter.setMaxListeners() to increase limit');
+                            `${existing.length} ${type.toString()} listeners ` +
```

Comment:
This isn't safe since `type` can currently be any type of value. So `null.toString()` will fail for example. It will probably be better to just concatenate with an empty string.